shared/params/config.go

```diff
@@ -10,29 +10,32 @@ import (
 // BeaconChainConfig contains constant configs for node to participate in beacon chain.
 type BeaconChainConfig struct {
 	// Misc constants.
-	ShardCount               uint64 `yaml:"SHARD_COUNT"`                 // ShardCount is the number of shard chains in Ethereum 2.0.
-	TargetCommitteeSize      uint64 `yaml:"TARGET_COMMITTEE_SIZE"`       // TargetCommitteeSize is the number of validators in a committee when the chain is healthy.
-	MaxIndicesPerAttestation uint64 `yaml:"MAX_INDICES_PER_ATTESTATION"` // MaxIndicesPerAttestation is used to determine how many validators participate in an attestation.
-	MinPerEpochChurnLimit    uint64 `yaml:"MIN_PER_EPOCH_CHURN_LIMIT"`   // MinPerEpochChurnLimit is the minimum amount of churn allotted for validator rotations.
-	ChurnLimitQuotient       uint64 `yaml:"CHURN_LIMIT_QUOTIENT"`        // ChurnLimitQuotient is used to determine the limit of how many validators can rotate per epoch.
-	BaseRewardsPerEpoch      uint64 `yaml:"BASE_REWARDS_PER_EPOCH"`      // BaseRewardsPerEpoch is used to calculate the per epoch rewards.
-	ShuffleRoundCount        uint64 `yaml:"SHUFFLE_ROUND_COUNT"`         // ShuffleRoundCount is used for retrieving the permuted index.
+	ShardCount                uint64 `yaml:"SHARD_COUNT"`                  // ShardCount is the number of shard chains in Ethereum 2.0.
+	TargetCommitteeSize       uint64 `yaml:"TARGET_COMMITTEE_SIZE"`        // TargetCommitteeSize is the number of validators in a committee when the chain is healthy.
+	MaxIndicesPerAttestation  uint64 `yaml:"MAX_INDICES_PER_ATTESTATION"`  // MaxIndicesPerAttestation is used to determine how many validators participate in an attestation.
+	MinPerEpochChurnLimit     uint64 `yaml:"MIN_PER_EPOCH_CHURN_LIMIT"`    // MinPerEpochChurnLimit is the minimum amount of churn allotted for validator rotations.
+	ChurnLimitQuotient        uint64 `yaml:"CHURN_LIMIT_QUOTIENT"`         // ChurnLimitQuotient is used to determine the limit of how many validators can rotate per epoch.
+	BaseRewardsPerEpoch       uint64 `yaml:"BASE_REWARDS_PER_EPOCH"`       // BaseRewardsPerEpoch is used to calculate the per epoch rewards.
+	ShuffleRoundCount         uint64 `yaml:"SHUFFLE_ROUND_COUNT"`          // ShuffleRoundCount is used for retrieving the permuted index.
+	SecondsPerDay             uint64 `yaml:"SECONDS_PER_DAY"`              // SecondsPerDay defines how many seconds are in a given day.
+	MaxValidatorsPerCommittee uint64 `yaml:"MAX_VALIDATORS_PER_COMMITTEE"` // MaxValidatorsPerCommittee defines the upper bound of the size of a committee.
+	MinGenesisActiveValidatorCount uint64 `yaml:"MIN_GENESIS_ACTIVE_VALIDATOR_COUNT"` // MinGenesisActiveValidatorCount defines how many validator deposits needed to kick off beacon chain.
+	MinGenesisTime            uint64 `yaml:"MIN_GENESIS_TIME"`             // MinGenesisTime defines the lower bound of the genesis time.
```

Comment:
Please add more to the comment here. The comment current restates the variable name in a different way. How could the genesis time have a minimum value?